### PR TITLE
EN: Tutorial Quests (Don't merge)

### PR DESCRIPTION
**Pull request by Aida-Enna**

### CODE REVIEW

File: Tutorial/Training Quests/tr_010002.csv [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-tut_win_text100#0,"""<c 00ffff>敵からの攻撃を受ける直前に、<br>タイミングよく『ガード』を行うことで<br>『ジャストガード』が発生し、ダメージを０に<br>することができます。<br>ジャストガードは、ＰＰの消費もなく使えます。<c><br><br>さらに、敵にガードアーツによる反撃ダメージを<br>与えることができる近接戦闘において、<br>優秀なスキルです。"""
+text01#0,"""By <red>timing your guard<c>, you can negate an enemy's attack without using any PP."""
+text02#0,"""Watch the enemy's movement, and calmly guard against it."""
+text101#0,"""Task 2：<yellow>Do 3 Just Guards with your sword"""
+train_title#0,"""Task 2：<yellow>Do 3 Just Guards with your sword"""
+tut_title#0,"""Performing a Just Guard"""
+text100#0,"""・Guard using <%WeaponAction> just as<br>the enemy attacks."""
+text100_3b#0,"""・Guard using <%MainPaletteC> just as<br>the enemy attacks."""
+tut_win_title#0,"""About Just Guard"""
+tut_win_text100#0,"""<c 00ffff>Just before receiving damage from an enemy attack, if you guard at the right moment, you'll perform a Just Guard. This will negate the enemy damage without using PP.<c><br><br>In addition, this counterattack damages the enemy. In a close promity battle, it's an excellent skill to use."""
```
Comment (alama) [inline Tutorial/Training Quests/tr_010002.csv]: promity? or proximity?

File: Tutorial/Training Quests/tr_010008.csv [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+text03#0,"""You can accumulate gears by <red>fully completing a normal attack or guarding<c>. Try it out!"""
+text04#0,"""Let's charge the <red>Partizan Gear to level 1<c>, then try using a Photon Art."""
+text101#0,"""Task 5: <yellow>Activate a Photon Art at Gear level 1<c>"""
+train_title#0,"""Task 5: <yellow>Activate a Photon Art at Gear level 1<c>"""
+tut_title#0,"""Using Partizan Gear"""
+text100#0,"""・Use Photon Arts or full complete<br>\u3000normal attacks to build level 1 gear<br>・Use a Photon Art with <%PhotonArts>"""
+text100_3b#0,"""・Use Photon Arts or full complete<br>\u3000normal attacks to build level 1 gear<br>・Use a Photon Art with <%PhotonArts>"""
+tut_win_title#0,"""About Partizan Gear"""
+tut_win_text100#0,"""Class Skill [Partizan Gear]<br>This is a skill to increase the performance of your partizan weapon.<br><br>By using Photon Arts and fully completing normal attacks, your gear will rise and accumulate charges.<br><br>When using Photon Arts, a gear charge is consumed in exchange for increasing the power or range of the Photon Art."""
```
Comment (alama) [inline Tutorial/Training Quests/tr_010008.csv:12]: we should add `partizan` to the weapon word list

File: Tutorial/Training Quests/tr_010002.csv (hunk 1 above)
Comment (Aida-Enna) [reply Tutorial/Training Quests/tr_010002.csv]: danke

File: Tutorial/Training Quests/tr_030009.csv [hunk 3]
```diff
@@ -1,4 +1,4 @@
 name01#0,"""Xiera"""
-11_voice_train_04003#0,"""おつかれさまでした！"""
-11_voice_train_04004#0,"""これで「フォースの基礎」の<br>訓練終了です。<br>他の訓練もぜひ受けてみてくださいね。"""
+11_voice_train_04003#0,"""Congratulations!"""
+11_voice_train_04004#0,"""This is the end of the [REPLACEME Basics] training. Please try the other training lessons as well!"""
```
Comment (alama) [inline Tutorial/Training Quests/tr_030009.csv:3]: REPLACEME?

**Pull request closed without merging**